util/ci_container_inner.sh

```diff
@@ -6,6 +6,8 @@ export WPT_PATH=/web-platform-tests
 
 git clone --depth 1 https://github.com/w3c/web-platform-tests $WPT_PATH
 
+sudo chown -R $(whoami):$(whoami) $HOME
```

Comment:
Prefer `$(id -u $USER):$(id -g $USER)` in case group not named after user.

Reply:
Done, thanks